src/components/BaseKey.vue

```diff
@@ -322,6 +322,7 @@ export default {
 @import '../scss/sp-pbt';
 @import '../scss/mt3-pbt';
 @import '../scss/mt3-abs';
+@import '../scss/dcs-abs';
```

Comment:
The Signature Plastics / Pimp My Keyboard color list is at `src/scss/sp-abs.scss`.

```suggestion
```